### DISCUSSION TITLE
Omrin werkt niet meer in mijn lua script

### DISCUSSION THREAD

**Discussion by spanjard** [jvanderzande/GarbageCalendar#34] [General]
Beste GarbageCalendar,

sinds kort werkt omrin niet meer in de gemeente weststellingwerf maar zo te zien heeft dit eerder met de webpagina te maken van omrin die recentelijk vernieuwd is. Heeft u mogelijk een oplossing hiervoor? ik zou het script graag willen corrigeren.

**Comment by jvanderzande** [comment]
EDIT: Ik denk dat het een algemeen probleem is met OMRIN, want de 2 test adressen die ik al heb gaan ook fout.
Dus zal even kijken.

**Comment by jvanderzande** [comment]
De Website is totaal veranderd en is niet een eenvoudige fix. Als ik naar de source van de Webpagina kijk wordt het lastig alle info te verzamelen en ik weet nog niet precies welke CURL requests te doen om uberhaubt de source in een bestand te krijgen.

**Comment by spanjard** [comment]
Hallo Jos,

Nou dan zijn we het met elkaar eens dat de webpagina niet meer uitgelezen
kan worden sinds de verandering van Omrin en dat hierdoor het script stuk
is gegaan.
alleen ben jij vele malen script technischer dan ik dat ben dus ik hoop dat
je een weg vindt om het weer werkend te krijgen.
de source is super rommelig nu ze die uit vouw pijltjes ingebouwd hebben
idd.


On Tue, 31 Jan 2023 at 20:28, Jos ***@***.***> wrote:

> De Website is totaal veranderd en is niet een eenvoudige fix. Als ik naar
> de source van de Webpagina kijk wordt het lastig alle info te verzamelen en
> ik weet nog niet precies welke CURL requests te doen om uberhaubt de source
> in een bestand te krijgen.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/jvanderzande/GarbageCalendar/discussions/34#discussioncomment-4833375>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACC2IVMDULOTIKV6YDXB7ULWVFRXLANCNFSM6AAAAAAUMYG73M>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 


Met vriendelijke groeten,

Raul Muñoz-Repko

**Comment by jvanderzande** [comment]
We hebben de API definities van OMRIN nodig om dit weer goed te laten werken. Ik heb deze gevonden, maar nog geen voor het ophalen van de afval kalender informatie:
```
https://www.omrin.nl/api/postcodeapi/1111xx/99
https://www.omrin.nl/api/calendarpdf?zipCode=1111xx&houseNumber=99&year=2023

```

**Comment by jvanderzande** [comment]
Ik heb omrin weer werkend. Laat me weten als je wilt testen.

**Comment by spanjard** [comment]
Ja graag!

**Reply by jvanderzande** [reply]
Ik had je al een email reply met update gestuurd vorige week. Misschien mij even een emailadres doorgeven zodat ik de test versie kan sturen?

**Reply by spanjard** [reply]
> Ik had je al een email reply met update gestuurd vorige week. Misschien mij even een emailadres doorgeven zodat ik de test versie kan sturen?

ik kon geen oplossing in me mail vinden ik pm je even.

**Comment by jvanderzande** [comment]
Omrin werkt weer met commit c5ca47f in Master.
